bucket/twilio-cli.json

```
@@ -0,0 +1,46 @@
+{
+    "version": "5.3.0",
+    "description": "Twilio CLI. Unleash the power of Twilio from your command prompt.",
+    "homepage": "https://github.com/twilio/twilio-cli",
+    "license": {
+        "identifier": "MIT",
+        "url": "https://github.com/twilio/twilio-cli/raw/main/LICENSE"
+    },
```

```suggestion
    "license": "MIT",
```
For well-known licenses, we this can be simplified.